docs/Deployment/webdeployment.md

```diff
@@ -0,0 +1,32 @@
+# Introduction to Web Deployments.
+
+We will take you through possible `Web Deployments` on this page.
+We will keep updating the list as more providers offer BTCPay server as a service. 
```

Comment:
We should explain what web deployment is here "services who offer managed BTCPay Server hosting..."

Reply:
Agree. Will write a little more to explain. 
Thanks.